smoke/Makefile

```diff
@@ -15,9 +15,11 @@ build:
 # NYDUS_NYDUSIFY=/path/to/latest/nydusify \
 # SKIP_CASES=compressor=lz4_block,fs_version=5 \
 # make test
+#$(TESTS)
+#NativeLayer
 test: build
 	golangci-lint run
-	sudo -E ./smoke.test -test.v -test.timeout 10m -test.parallel=16 -test.run=$(TESTS)
+	sudo -E ./smoke.test -test.v -test.timeout 10m -test.parallel=16 -test.run=TestAPI
```

Comment:
Please don't modify the test cases, we need to run all tests with `make test` in somke test.

Reply:
See https://github.com/dragonflyoss/nydus/blob/master/.github/workflows/smoke.yml#L163C5-L184C32.

Reply:
Thank you for pointing out the issues. I will try to address them